<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tela_vendedor</class>
 <widget class="QMainWindow" name="tela_vendedor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Área do vendedor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(31, 31, 31);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Área do vendedor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 255, 255);</string>
    </property>
    <property name="text">
     <string>Digite o Código
 do Funcionário:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vendedor_codigo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
background-color: rgb(67, 67, 67);
border-radius:10px</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_vendedor">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(85, 255, 255);
border-radius:15px</string>
    </property>
    <property name="text">
     <string>Entrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../background/teste.qrc"/>
 </resources>
 <connections/>
</ui>
